vertica_python/errors.py

```diff
@@ -114,7 +114,8 @@ def __init__(self, error_response, sql):
         self.sql = sql
         ProgrammingError.__init__(self,
                                   "{0}, SQL: {1}".format(error_response.error_message(),
-                                                         repr(self.one_line_sql())))
+                                                         repr(self.one_line_sql())),
+                                  error_response.attributes)
```

Comment:
You add one more argument here but doesn't change the definition of `ProgrammingError.__init__()`.

Reply:
`ProgrammingError` inherited (via intermediate classes) from the `BaseException` without an `__init__()` overriding and includes `args` already: https://docs.python.org/3/library/exceptions.html#BaseException